packages/xo-web/src/common/xo/index.js

```diff
@@ -477,6 +477,25 @@ subscribeHostMissingPatches.forceRefresh = host => {
   }
 }
 
+const proxiesApplianceUpdaterState = {}
+export const subscribeProxiesApplianceUpdaterState = (proxyId, cb) => {
+  if (proxiesApplianceUpdaterState[proxyId] === undefined) {
+    proxiesApplianceUpdaterState[proxyId] = createSubscription(() => getProxyApplianceUpdaterState(proxyId))
+  }
+  return proxiesApplianceUpdaterState[proxyId](cb)
+}
+subscribeProxiesApplianceUpdaterState.forceRefresh = proxyId => {
+  if (proxyId === undefined) {
+    forEach(proxiesApplianceUpdaterState, subscription => subscription.forceRefresh())
+    return
+  }
+
+  const subscription = proxiesApplianceUpdaterState[proxyId]
+  if (subscription !== undefined) {
+    subscription.forceRefresh()
+  }
+}
+
```

Comment:
The proxy page should also use this subscription, so that refreshing one of them refresh both.

Current version appears to have an issue: after upgrading a proxy, the menu still indicates that some proxies need to be upgraded.